src/workspaces/ShareWorkspaceModal/Collaborator.test.ts

```diff
@@ -280,4 +280,45 @@ describe('a Collaborator component', () => {
       }
     });
   });
+
+  describe('hides the Can Compute option for collaborators with Reader access', () => {
```

Comment:
Super Nit: The describe is to group tests,  so it makes a bit more sense if it's a more general description. Something like 'the visibility of the cam compute option'. Then the individual test cases elaborate what part of it they're testing. It reads nicely on test reports.